### ISSUE TITLE
QSelect maxlength is not passed

### ISSUE THREAD

**Issue by DarkXanteR** [quasarframework/quasar#6619]
**Describe the bug**
Attribute `maxlength` set on QSelect isn't passed to the input field.

**Codepen/jsFiddle/Codesandbox (required)**
https://codepen.io/darkxanter/pen/gOpzpRE

**Expected behavior**
Attribute `maxlength` must be passed to input field.

**Comment by mateonunez** [comment]
That's because `maxlength` is not a property of QSelect component.
This is a feature request, not a bug.

**Comment by DarkXanteR** [comment]
In QSelect doc says: 

> All the attributes set on QSelect that are not in the list of props in the API will be passed to the native input field used (please check use-input prop description first to understand what it does) for filtering / autocomplete / adding new value. Some examples: autocomplete, placeholder.
https://quasar.dev/vue-components/select#Native-attributes-with-use-input

And it is not work for this input field attribute.

**Comment by mateonunez** [comment]
I found the problem: the conflict is when QSelect is rendering `input` element, using `use-input`. In `input` element are not passing correctly `$attrs` -> `input native props`.

I closed my PR because was not the right solution. One possible solution could be to transform `input native props` into an object and pass it into QSelect component, then `__getInput()` can render it.

> Component.vue
```vue
<template>
  <q-select
    use-input
    fill-input
    hide-selected
    :attrs="attrs"
  >
    <template v-slot:no-option>
      <q-item>
        <q-item-section class="text-grey">
          No results
        </q-item-section>
      </q-item>
    </template>
  </q-select>
</template>
<script>
export default{
  ...,
  data() {
    return {
      ...,
      attrs: {
        max: 5,
        type: 'number',
        step: '0.01'
      }
    }
  }
}
</script>
```

> QSelect.js#__getInput(h)
```js
__getInput (h, fromDialog) {
      ...,

      return h('input', {
        ...,
        attrs: {
          ...this.$attrs.attrs,
        },
        ...,
      })
    },
```

In the next days I'll make a new PR.

**Comment by rstoenescu** [comment]
Enhancement will be available in "quasar" v1.9.11.
